experimental/kubernetes-v2/config/clouddriver.yml

```diff
@@ -0,0 +1,92 @@
+server:
```

Comment:
That, cleaning up script behavior (warning on deleting data stores, not printing errors on being unable to create existing resources, waiting on results to complete rather than calling `sleep`), and improving the story around persistent data stores are things I'm saving for the next PR.